kolibri/core/assets/src/components.js

```diff
@@ -10,13 +10,23 @@ logging.info('Component views loaded!');
 
 var AbstractTextInput = Mn.ItemView.extend({
     template: function(serialized_model) {
-        var template_html;
-        if (this.model.get('enabled')) {
-            template_html = '<input type="' + this.input_tag_type + '" placeholder="Search here!">';
-        } else {
-            template_html = '<input type="' + this.input_tag_type + '" disabled placeholder="Search here!">';
+        var template_html = '<input type="' + this.input_tag_type +'" ';
+
+        logging.info(typeof template_html);
```

Comment:
Try to avoid committing debug statements like this. You'll want to remove it before we merge anything,